### PR TITLE
added two new rows(pick up date, is decline)

### PR DESCRIPTION
**Pull request by aalassv**
Adding the new columns to the main table two handle if is the bike request is decline and pick up date.

### CODE REVIEW

File: client/src/Component/MainTable.js [hunk 1]
```diff
@@ -64,6 +64,8 @@ const MainTable = () => {
 						<th scope="col">DOB</th>
 						<th scope="col">Postcode</th>
 						<th scope="col">Referring agency</th>
+						<th scope="col">Pick up date</th>
+						<th scope="col">Decline</th>
```
Comment (bonboh) [inline client/src/Component/MainTable.js:68]:
```suggestion
						<th scope="col">Declined</th>
```

File: client/src/Component/MainTable.js [hunk 2]
```diff
@@ -86,6 +88,8 @@ const MainTable = () => {
 								<td>{new Date(client.date_of_birth).toLocaleDateString()}</td>
 								<td>{client.postcode}</td>
 								<td>{client.referring_agency}</td>
+								<td>{new Date(client.pick_up_date).toLocaleDateString()}</td>
+								<td>{client.is_decline}</td>
```
Comment (bonboh) [inline client/src/Component/MainTable.js:92]:
```suggestion
								<td>{client.is_declined}</td>
```
This should match the database column name in [create_database.sql](https://github.com/CodeYourFuture/scot-callback-cats/blob/main/server/scripts/create_database.sql#L29)

**Review by bonboh — CHANGES_REQUESTED** [review]
Good job @aalassv! Just a couple of small suggestions

File: client/src/Component/MainTable.js (hunk 1 above)
Comment (aalassv) [reply client/src/Component/MainTable.js:68]: Thanks Coung.

File: client/src/Component/MainTable.js (hunk 2 above)
Comment (aalassv) [reply client/src/Component/MainTable.js:92]: Already changed thanks again.

**Review by bonboh — APPROVED** [review]
@aalassv LGTM!

**Pull request merged**